bindings/pyroot_experimental/pythonizations/src/FacadeHelpers.cxx

```diff
@@ -0,0 +1,46 @@
+// Author: Enric Tejedor CERN  04/2020
+
+/*************************************************************************
+ * Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+// Bindings
+#include "FacadeHelpers.hxx"
+
+// Cppyy
+#include "LowLevelViews.h"
+
+// ROOT
+#include "RtypesCore.h"
+#include "ROOT/RConfig.hxx"
+
+//////////////////////////////////////////////////////////////////////////
+/// \brief Get a buffer starting at a given address
+/// \param[in] self Always null, since this is a module function.
+/// \param[in] addr Address to create buffer from
+///
+/// Returns a cppyy LowLevelView object on the received address, i.e. an
+/// indexable buffer starting at that address.
+PyObject *PyROOT::CreateBufferFromAddress(PyObject * /* self */, PyObject *addr)
+{
+   if (!addr) {
+      PyErr_SetString(PyExc_RuntimeError, "Unable to create buffer from invalid address");
+      return NULL;
+   }
+
+   Long64_t cAddr = PyLong_AsLongLong(addr);
```

Comment:
You don't really need the `Long64_t` here, you could just use a `long long`, which is always 64 bit, also on the 32 bit machines. But just a minor detail, doesn't hurt that way. You could get rid of the `RtypesCore.h` header that way.

Reply:
Ok but, then in line 42 and 44, you would just use `long long` (64 bit) and `int` (32 bit) too, instead of the Rtypes? I need the header for those too.

Reply:
Would be possible, I guess. As you like!